ci/test_cpp.sh

```diff
@@ -8,20 +8,38 @@ trap "EXITCODE=1" ERR
 set +e
 
 # Run libcudf and libcudf_kafka gtests from libcudf-tests package
-rapids-logger "Run gtests"
-
-cd $CONDA_PREFIX/bin/gtests/libcudf/
 export GTEST_OUTPUT=xml:${RAPIDS_TESTS_DIR}/
 
+pushd $CONDA_PREFIX/bin/gtests/libcudf/
+rapids-logger "Run libcudf gtests"
 ctest -j20 --output-on-failure
-
 SUITEERROR=$?
+popd
 
 if (( ${SUITEERROR} == 0 )); then
-    cd $CONDA_PREFIX/bin/gtests/libcudf_kafka/
+    pushd $CONDA_PREFIX/bin/gtests/libcudf_kafka/
+    rapids-logger "Run libcudf_kafka gtests"
     ctest -j20 --output-on-failure
     SUITEERROR=$?
+    popd
+fi
+
+# Ensure that benchmarks are runnable
+pushd $CONDA_PREFIX/bin/benchmarks/libcudf/
+rapids-logger "Run tests of libcudf benchmarks"
+
+if (( ${SUITEERROR} == 0 )); then
+    # Run a small Google benchmark
+    ./MERGE_BENCH --benchmark_filter=/2/
+    SUITEERROR=$?
+fi
+
+if (( ${SUITEERROR} == 0 )); then
+    # Run a small nvbench benchmark
+    ./STRINGS_NVBENCH --run-once --benchmark 0
```

Comment:
```suggestion
    ./STRINGS_NVBENCH --run-once --benchmark 0 -d 0
```
I would recommend specifying a single device to minimize the number of runs in case multiple GPUs are available.

Reply:
Good idea. I'll use the long form of the flag for consistency.
```suggestion
    ./STRINGS_NVBENCH --run-once --benchmark 0 --devices 0
```